syntax = "proto3";

package api.user.request;

message CreateConnection {
    uint64 user_id = 1;
}
